services/checkin/models/user_roles.go

```diff
@@ -0,0 +1,6 @@
+package models
+
+type UserRoles struct {
+	ID    string   `bson:"id"    json:"id"`
```

Comment:
We shouldn't have `bson` tags anywhere in the codebase anymore. If there's somewhere they should be removed.